DESCRIPTION

```diff
@@ -1,17 +1,17 @@
 Type: Package
 Package: opencage
-Title: Interface to the OpenCage API
-Version: 0.1.4.9001
+Title: Geocode with the OpenCage API
+Version: 0.1.4.9002
```

Comment:
clever, better for SEO!

Reply:
That was the idea. IIRC I was indeed searching for the package somewhere and was surprised I could not find it.